### DISCUSSION TITLE
Add support for Headers object

### DISCUSSION THREAD

**Discussion by andreynovikov** [faisalman/ua-parser-js#763] [Ideas]
In some Node environments `request.headers` is not a dictionary, but a `Headers` object: https://developer.mozilla.org/en-US/docs/Web/API/Headers
It would be great if the library can handle it natively without the need to convert it to dictionary.

**Comment by faisalman** [comment]
Hi, thank you for your suggestion. I've just added it in https://github.com/faisalman/ua-parser-js/commit/d5edcbae12ffe7025199d8839e2fb5c78cdb9e38

**Comment by andreynovikov** [comment]
Works great, thank you!
